src/Subscribers/DoctrineEncryptSubscriber.php

```diff
@@ -175,19 +167,14 @@ public function postLoad(LifecycleEventArgs $args)
      */
     public function getSubscribedEvents()
```

Comment:
I meant the return type hint could be added to @driesvints PR (which targets PHP 7.1)